custom_components/temperature_feels_like/sensor.py

```diff
@@ -304,9 +313,9 @@ def _get_wind_speed(self, entity_id: Optional[str]) -> Optional[float]:
 
     async def async_update(self):
         """Update sensor state."""
-        temp = self._get_temperature(self._temp)  # °C
-        humd = self._get_humidity(self._humd)  # %
-        wind = self._get_wind_speed(self._wind)  # m/s
+        self._temp_val = temp = self._get_temperature(self._temp)  # °C
+        self._humd_val = humd = self._get_humidity(self._humd)  # %
+        self._wind_val = wind = self._get_wind_speed(self._wind)  # m/s
 
         _LOGGER.debug("Temp: %s °C  Hum: %s %%  Wind: %s m/s", temp, humd, wind)
```

Comment:
Your code makes uninitialized variables. Please update also this line..

Reply:
I'm not sure what you mean by 'makes uninitialized variables' - `self._temp_val = temp = self._get_temperature(self._temp)` assigns to both `self._temp_val` and `temp` variables. `self._*_val` variables are initialized in `__init__`

Reply:
Oh sorry, my mistake.  Was tired and did not notice the second assignments.